### PR TITLE
Changes to pydfnworks install instructions.

### PR DESCRIPTION
**Pull request by aidanstansberry**
Changes to install instructions for pydfnworks using pip to avoid future issues with past install method. Just the github docs and readme files.

### CODE REVIEW

File: pydfnworks/pydfnworks/dfnGen/meshing/udfm/upscale.py
Comment (hymanjd) [inline pydfnworks/pydfnworks/dfnGen/meshing/udfm/upscale.py:1]: What are these changes from?

File: README.md
Comment (hymanjd) [inline README.md:1]: Did you add a requirements.txt file as well?

**Review by hymanjd — CHANGES_REQUESTED** [review]
Unclear about a few things. See comments.

File: pydfnworks/pydfnworks/dfnGen/meshing/udfm/upscale.py
Comment (aidanstansberry) [reply pydfnworks/pydfnworks/dfnGen/meshing/udfm/upscale.py:1]: I accidentally committed some changes that I must have made on my fork at some point, I replaced the file with the file from lanl/dfnworks, so there shouldn't be any differences here. Just shows up in the commit log

File: README.md
Comment (aidanstansberry) [reply README.md:1]: Yep, should be in there

File: pydfnworks/pydfnworks/dfnGen/meshing/udfm/upscale.py
Comment (aidanstansberry) [reply pydfnworks/pydfnworks/dfnGen/meshing/udfm/upscale.py:1]: If you look at the two different merges you can see the file is changed to the accidental commit, and then changed back in the second one

File: README.md
Comment (hymanjd) [reply README.md:1]: looks good

File: pydfnworks/pydfnworks/dfnGen/meshing/udfm/upscale.py
Comment (hymanjd) [reply pydfnworks/pydfnworks/dfnGen/meshing/udfm/upscale.py:1]: okay. Thanks

File: pydfnworks/requirements.txt
Comment (hymanjd) [inline pydfnworks/requirements.txt:1]: This should contain the dependency packages like numpy, scipy, networkx, etc? correct?

**Review by hymanjd — APPROVED** [review]

**Review by hymanjd — APPROVED** [review]

**Review by hymanjd — APPROVED** [review]

**Comment by hymanjd** [comment]
Thanks Aidan.

File: pydfnworks/requirements.txt
Comment (aidanstansberry) [reply pydfnworks/requirements.txt:1]: Yes, as I understand it. Right now its just set up to install pydfnworks, but we can add other specific package versions as well

Comment (hymanjd) [reply pydfnworks/requirements.txt:1]: I think it'll just pull what's in the release.py file

Comment (aidanstansberry) [reply pydfnworks/requirements.txt:1]: Yes, that is correct

**Pull request merged**